package.json

```diff
@@ -103,6 +103,7 @@
     "d3-scale": "1.0.3",
     "d3-shape": "1.0.3",
     "d3-time": "1.0.2",
+    "react-dimensions": "2.0.0-alpha1",
```

Comment:
I think we're fine since we only support Chrome for the dataviz so any issues should easily come up during QA, although perhaps I'll make a note to test in Chrome on Windows as well. I've also looked at the repo's GitHub issues, and nothing huge stands out. v2 has breaking changes and is basically a better approach, and for our use case has produced no problems so far.